sdk/core/azure-core/inc/http/policy.hpp

```diff
@@ -72,26 +74,31 @@ namespace Azure { namespace Core { namespace Http {
 
   struct RetryOptions
   {
-    int16_t MaxRetries = 5;
-    int32_t RetryDelayMsec = 500;
+    int32_t MaxRetries = 3;
+
+    std::chrono::duration<int64_t, std::milli> RetryDelay = std::chrono::seconds(4);
```

Comment:
There is a helper type `std::chrono::milliseconds`.

ref https://en.cppreference.com/w/cpp/chrono/duration

Reply:
Thank you, I'll check it out.

Reply:
Done.